src/uct/ib/ud/base/ud_iface.c

```diff
@@ -575,6 +584,11 @@ ucs_config_field_t uct_ud_iface_config_table[] = {
      "Enable checking destination GID for incoming packets of Ethernet network.\n"
      "Mismatched packets are silently dropped.",
      ucs_offsetof(uct_ud_iface_config_t, dgid_check), UCS_CONFIG_TYPE_BOOL},
+
+    {"MAX_WINDOW", UCS_PP_MAKE_STRING(UCT_UD_CA_MAX_WINDOW),
+     "Max congestion avoidance window.",
```

Comment:
plz document the valid range for the window, since you print error when it is out of "valid range"